ODataConnectedService/src/ViewModels/ConfigODataEndpointViewModel.cs

```diff
@@ -77,6 +77,8 @@ private string GetMetadata(out Version edmxVersion)
                 }
             }
 
+            ServicePointManager.ServerCertificateValidationCallback += new System.Net.Security.RemoteCertificateValidationCallback((s, ce, ch, ssl) => true);
```

Comment:
Is this meant for test/development environment only, or have you encountered issues outside of the development process?

In either case, we should add some guards inside the lambda. We should add a boolean such as isDevelopment which returns true as presented in your code. We should also add validation when isDevelopment == false. I would also recommend using more descriptive variable names. Below is my comprehensive suggestion: